Automattic/jetpack
# CRM: Update Automations category labels: translate and capitalize automation

Fixes https://github.com/Automattic/zero-bs-crm/issues/3280

## Proposed changes:

This PR just updates the category names in the steps, translating and capitalizing them with the current implementation (string acting as a label). We are not going to use categories in this phase of the project, and we will probably create the Category class, which allows us to have more flexibility, like adding a proper slug, label, color, and icon.

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
N/A

## Testing instructions:
- Check the code and the category names. They should be translated and capitalized.
- Run the test and check that they pass.
